.deepsource.toml

```diff
@@ -6,6 +6,10 @@ exclude_patterns = [
   "ci/**",
   "InvenTree/**/migrations/*.py"
 ]
+test_patterns = [
+  "test_.*.py"
```

Comment:
Probably wants to also include "tests.py" - we have a few instances of that

Reply:
forgot about them, will amend. Thx